src/cli/start.rs

```diff
@@ -43,6 +45,13 @@ impl CliCommand for Start {
             log::info!("{}", error);
         }
         // Open the app
-        open::that("http://localhost:4000").unwrap();
+        println!(
+            "{}",
+            format!(
+                "{}{}",
+                "Application available at http://localhost:",
+                docker_compose::DEVTOOL_PORT
+            )
+        );
```

Comment:
Let's do it simpler ^^

```suggestion
        println!("Application available at http://localhost:{}", docker_compose::DEVTOOL_PORT);
```

Reply:
ok, it's that easy